### PR TITLE
group create

### PR DESCRIPTION
**Pull request by Chirag-Khatri-CK**

### CODE REVIEW

File: src/views/Group/Group.vue [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    <h3 class="p-3 icon-title h6 mb-0 font-weight-bold">Invite People</h3>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "Group",
+  data() {
+    return {
+      openSettings: 0,
```
[the remark this answers is not in the document]
Comment (Chirag-Khatri-CK) [reply src/views/Group/Group.vue:38]: when we click on settings icon.

File: src/views/Group/CreateGroup.vue [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+
+    createGroup() {
+      const { groupName, groupDescription, members, isPaid } = this;
+      const { dispatch } = this.$store;
+
+      if (!groupName) {
+        dispatch(
+          "alertStore/error",
+          stringFormat(
+            config.messagingConfig.messages.error.field_error,
```
[the remark this answers is not in the document]
Comment (Chirag-Khatri-CK) [reply src/views/Group/CreateGroup.vue:203]: done sir

File: src/services/group.service.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
+import { authHeader, stringFormat } from '@/helpers';
+import { config } from "@/configurations";
+import axios from "axios";
+
+export const groupService = { searchProfile , addGroup }
```
[the remark this answers is not in the document]
Comment (Chirag-Khatri-CK) [reply src/services/group.service.js:5]: done sir

File: src/services/group.service.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    const url = `${config.commonConfig.$apiUrl}/${config.groupConfig.api.createGroup.endpoint}`;
+    return axios
+        .post(
+            url,
+            createGroupBody,
+            { headers: headers },
+        )
+        .then((response) => {
+            const group_info = response.data
+            console.log(group_info);
```
[the remark this answers is not in the document]
Comment (Chirag-Khatri-CK) [reply src/services/group.service.js:36]: removed log

**Review by chayan007 — APPROVED** [review]

**Pull request merged**